lib/events/codes.go

```diff
@@ -271,6 +277,9 @@ const (
 	AccessRequestUpdateCode = "T5001I"
 	// ResetPasswordTokenCreateCode is the token create event code.
 	ResetPasswordTokenCreateCode = "T6000I"
+	// TrustedClusterTokenCreateCode is the event code for
+	// creating new join token for a trusted cluster.
+	TrustedClusterTokenCreateCode = "T7002I"
```

Comment:
Should this be `T7000I`?

Reply:
In my other PR, I have this:
```
	// TrustedClusterCreateCode is the event code for creating a trusted cluster.
	TrustedClusterCreateCode = "T7000I"
	// TrustedClusterDeleteCode is the event code for removing a trusted cluster.
	TrustedClusterDeleteCode = "T7001I"
```
But with that said, I have to refactor all the codes I made up, because I belatedly read how codes were formed: https://github.com/gravitational/gravity/blob/master/lib/ops/events/codes.go#L289

I am going to create a PR to refactor code and place the same documentation in teleport, after I finish merging my PRs.